File: data/c#/pattern_6/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        int k = 1;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            for (int j = 1; j <= num; j++)
            {
                Console.Write(k + " ");
                k++;
            }
            Console.Write("\n");
        }
    }
}

File: data/c/pattern_6/code.c
#include <stdio.h>

int main(void)
{
    int num;
    int k = 1;
    printf("Enter the number of rows and columns for the square: ");
    scanf("%d", &num);

    for (int i = num; i >= 1; i--)
    {
        for (int j = num; j >= 1; j--)
        {
            printf("%d ", k++);
        }
        printf("\n");
    }

    return 0;
}
